src/Pages/Online-Monday/MondayOnline.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./MondayOnline.module.css";
+import AddClassOnline from "../../Components/AddClassOnline/AddClassOnline";
+import NavigationBar from "../../Components/NavigationBar/NavigationBar";
+
+function MondayOnline(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <p className={styles.headline}>
+        <a href="./Online" className={styles.arrow}>
+          <span>←</span>
+        </a>
```

Comment:
Please consider adding an alternative text to be read out for more accessibility. It could use a `screen reader only` css class to hide it for people that can see the icon.